Thesis.tex

```diff
@@ -171,6 +171,26 @@ \section{Mentality concerning IP rights}
 
 \section{Globalisation \& increased complexity of rights management}
 \label{sec:GlobalRightsMgmt}
+The global development of industries over the past few decades had quite a significant impact related to patents. More companies from all over the world hold patents relevant to sometimes whole industry branches. The number of patents involved with modern products and services also reached a new scale. One example for this is the LTE \footcite{Long Term Evolution} standard for mobile communications that lead to several law suits between different technology companies over a number of years.
+
+\subsection{Patent magnitude}
+As technology gets more complex, more companies are involved in development of individual elements that make up standards, services and products of our modern age. And since new technologies are often derived from established ones it is no surprise that some modern products are related to astounding numbers of patents. As an example we can take a look at the 3GPP mobile communications standard. It is estimated that there are over 8000 patents considered essential for this industry standard, where approximately 90\% of those are split between 12 companies \parencite{Wiki3G}. It is clear that with such large pools of patents relevant to industry standards proper licensing is a complex matter.
+
+The follow-up standard LTE is also no exception. While it is not easy to find exact numbers on the size of the patent pool related to the standard, its effects were rather visible over the past few years. A number of lawsuits have been filed over standard relevant patents: Huawei versus ZTE \parencite{HuaweiVsZTE}, Apple versus Ericsson \parencite{AppleVsEricsson}, Huawei versus Samsung \parencite{HuaweiVsSamsung} and Huawei versus T-Mobile \parencite{HuaweiVsTmobile} to just name a few.
+
+\subsection{Cross-licensing}
+To limit the risk of patent infringement lawsuits many companies enter cross-licensing agreements. Parties involved in a cross-licensing agreement grant each other a license to exploit patents related to the subject matter of the agreement. Very often each party owns patents related to different aspects of a given product. Thus a cross-license maintains their freedom to bring new products to market \parencite{CrossLicense}.
```

Comment:
Ich würde hier use patents verwenden anstelle exploit patents. Exploit hat definitiv in der Mehrzahl der Bedeutungen einen negativ wertenden Beigeschmack